compiler-cli/src/build.rs

```diff
@@ -20,6 +20,18 @@ pub fn download_dependencies() -> Result<Manifest> {
 }
 
 pub fn main(options: Options, manifest: Manifest) -> Result<Built> {
+    compile(options, manifest, ModulesCompilation::CompileAll)
+}
+
+pub fn deps_only(options: Options, manifest: Manifest) -> Result<Built> {
+    compile(options, manifest, ModulesCompilation::IgnoreModules)
```

Comment:
This name was confusing to me- it says to ignore modules but it compiles all modules except for the root package. Something like this would make more sense to me at this level:

```rust
pub enum CompilationScope {
    AllModules
    OnlyDependencyModules
}

Reply:
Yeah that is way clearer, thanks!